parsers/CA_NS.py

```diff
@@ -97,7 +113,7 @@ def _get_ns_info(requests_obj, logger: Logger):
             )
 
         electricity_mix = {
-            gen_type: percent_value * load
+            gen_type: round(percent_value * load, 3)
```

Comment:
Not sure round is needed, the production_mix.add_value() will round any number to 6 decimals internally.